### PR TITLE
Some helpful aliases...

### PR DESCRIPTION
**Pull request by Xliff**

### CODE REVIEW

**Comment by Xliff** [comment]
```
<SmokeMachine> Xliff: if we are going to create aliases for `.^all.grep` and `.^all.map`, why not call they `.^grep` and `.^map`?
<SmokeMachine> Xliff: and, update is just one of the uses of `.^all.map`...
```

I don't mind adding aliases for those, either. The whole point is to given them useable names that describe the operations they can be used for.

`.^all.grep` and `.^all.map` are hard to remember where as .^where and .^update are easy, especially for new users. As they grow more familiar with Red, they can grow into the use of .^all.grep and .^all.map for more complex use cases.

**Comment by FCO** [comment]
Xliff, I’m sorry, but I think just the opposite... grep and map are Raku syntax. And .^all is just a way to return the ResultSeq... so, the new user have just know about the .^all, and use anything it would use with an array...

**Comment by Xliff** [comment]
@FCO I suggest you take a lesson from one of the talks jnthn gave. He said that as a language developer (and in a sense, you are a language developer), you have to be willing to torture yourself for the sake of the user.

In this situation, that means that we (and yes, in a sense I am a language developer, too) must think about things that make a users life easier.

You also need a little flexibility. Your resistance to an _alias _

File: lib/MetamodelX/Red/Model.pm6 [hunk 1]
```diff
@@ -244,6 +244,16 @@ multi method rs($, :$with! --> Red::ResultSeq) { $.rs-class.new.with: $with }
 multi method all($obj --> Red::ResultSeq)          { $obj.^rs }
 multi method all($obj, :$with! --> Red::ResultSeq) { $obj.^rs(:$with) }
 
+#| Alias for .^rs.grep
+method where($obj, Callable $filter --> Red::ResultSeq) {
+  $obj.^rs.grep($filter)
+}
+
+#| Alias for .^rs.map
+method update($obj, Callable $filter --> Red::ResultSeq {
```
Comment (FCO) [inline lib/MetamodelX/Red/Model.pm6]: there are no closing brackets here...

**Pull request closed without merging**